### ISSUE TITLE
Account number should be editable (Argentina)

### ISSUE THREAD

**Issue by fedegrc** [bisq-network/bisq#1543]
We need you to change the field "Número de cuenta" / "Account number" from read-only to editable for recently supported for Argentina:
- Cash Deposit
- National bank transfer
- Transfer with same bank
- Transfers with specific banks

Looks like that number doesn't follow a rigid rule, so banks are able to change the way they create it. It would be great if users could edit that field after it is auto-filled based on the CBU number. Additionally, special characters  '-' and '/' should be allowed.

In summary: 
- please don't remove the currently auto-filled
- allow users to change that value (at least '-' and '/' should be allowed)
- The change should be applied to Cash Deposit | National bank transfer | Transfer with same bank | Transfers with specific banks

https://user-images.githubusercontent.com/4233784/38172661-d7b47862-35b0-11e8-8438-cea521a1ac2c.png

Thank you!
